hikari/internal/data_binding.py

```diff
@@ -75,62 +76,131 @@
 Stringish = typing.Union[str, int, bool, undefined.UndefinedType, None, snowflakes.Unique]
 """Type hint for any valid that can be put in a StringMapBuilder"""
 
+JSONEncoder = typing.Callable[[typing.Union[JSONArray, JSONObject]], typing.Union[str, bytes]]
+"""Type hint for hikari-valid JSON encoders.
+
+A hikari-valid JSON encoder is one which will take in a JSON-ish object and output either `str`
```

Comment:
I think the phrasing `hikari-comptible` would be less awkward here. This isn't an approval-blocking issue to me tho.

Reply:
Changed!